internal/iruntime/total_memory_other.go

```diff
@@ -16,14 +16,8 @@
 
 package iruntime
 
-import (
-	"fmt"
-)
-
-var errTotalMemoryNotAvailable = fmt.Errorf("reading cgroups total memory is available only on linux")
-
 // TotalMemory returns total available memory.
 // This is non-Linux version that returns -1 and errTotalMemoryNotAvailable.
```

Comment:
This returns a `uint64` so it can't return `-1`.  It also doesn't return `errTotalMemoryNotAvailable`.

Reply:
Thanks for catching it.